docs/install/installing-istio.md

```diff
@@ -86,12 +86,14 @@ spec:
             - port: 15020
               name: status-port
             - port: 80
+              targetPort: 8080
               name: http2
             - port: 443
+              targetPort: 8443
               name: https
 EOF
 
-istioctl manifest apply -f istio-minimal-operator.yaml
+istioctl manifest install -f istio-minimal-operator.yaml
```

Comment:
I think it's just `istioctl install`? Unless there is an alias I am not familiar with

Reply:
Yeah, it seems `istioctl manifest install` was forgotten to remove. I am trying to change to remove it https://github.com/istio/istio/pull/27277 so we should use `istioctl install` instead.

Reply:
I removed the manifest in this PR